### ISSUE TITLE
Java Error with latest docker compose image

### ISSUE THREAD

**Issue by deeztek** [Ortus-Solutions/docker-commandbox#84]
Trying to launch latest Commandbox with docker compose file below:

```
 commandbox:
    image: ortussolutions/commandbox
    container_name: commandbox_app
    logging:
      options:
        max-size: 10m
    environment:
      TZ: "America/New_York"
      cfconfig_adminPassword: supersecretpass
      url_rewrites: "true"
    volumes:
      - app:/app #Application Directory
      - custom:/custom #Custom Directory
      - /scripts:/scripts
    restart: unless-stopped
    ports:
      - "8282:8080"
```

Getting the following error in container logs:

```
Exception in thread "main" java.lang.NumberFormatException: For input string: "default"
cat java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
at java.lang.Integer.parseInt(Integer.java:580)
at java.lang.Integer.parseInt(Integer.java:615)
at runwar.options.SiteOptions.errorPages(SiteOptions.java:419)
at runwar.options.ConfigParser.parseOptions(ConfigParser.java:476)
at runwar.options.ConfigParser.<init>(ConfigParser.java:34)
at runwar.Start.main(Start.java:37)
```

This used to work, now I"m getting Java errors. Not sure what the problem is. Some help would be appreciated.

Thanks

**Comment by jclausen** [comment]
@deeztek I would have to see your `server.json` contents to know what the issue is.  It is not in your compose declaration, but likely in something that has changed with CommandBox 6.

If you need to roll back, you can use `ortussolutions/commandbox:5.9.1` as your image, in the short term

**Comment by bdw429s** [comment]
This was a regression in CommandBox 6.0 which has been fixed in 6.1-rc

**Comment by jclausen** [comment]
Images are building now with an updated CommandBox version.
